procedures/auxtel/observation_procedures/Parabolic_Focus_Sequence.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>Instantiate</p><p> </p><p>I think it would be good to write a sentence or two about the reasoning for instantiating the CWFS script. It seems a bit odd to use a CWFS script to do a parabolic focusing :)</p><p>Consider something like:</p><p>In many cases it is desirable to perform a CWFS focus sequence for comparison. Therefore, we instantiate this script and provide the option to the user after the Parabolic focus data acquisition.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Cheers. Added. </p><p> </p>